micrometer-observation-test/src/test/java/io/micrometer/observation/tck/ObservationValidatorTests.java

```diff
@@ -42,8 +42,8 @@ void doubleStartShouldBeInvalid() {
             .hasMessage("Invalid start: Observation 'test' has already been started")
             .satisfies(exception -> assertThat(exception.toString()).matches(
                     "(?s)^io\\.micrometer\\.observation\\.tck\\.InvalidObservationException: Invalid start: Observation 'test' has already been started\n"
-                            + "START: app//io\\.micrometer\\.observation\\.tck\\.ObservationValidatorTests\\.lambda\\$doubleStartShouldBeInvalid\\$\\d+\\(ObservationValidatorTests\\.java:\\d+\\)\n"
```

Comment:
I don't understand why this changed based on the main code changes. I guess we don't expect anything is depending on this output format?

Reply:
Me neither, the JDK somehow prints a different stacktrace if you get it in different ways. No one should depend on this behavior (except our tests).